examples/firmware_update_example/src/xi_sft_client.c

```diff
@@ -135,7 +139,7 @@ void on_sft_message( xi_context_handle_t in_context_handle,
                                  params->message.temporary_payload_data_length );
 
             /* Figure out what the packet type is and then call the appropriate parser */
-            cb = cn_cbor_decode( payload_data, payload_length, 0 );
+            cb = cn_cbor_decode( payload_data, payload_length CBOR_CONTEXT_PARAM, NULL );

```

Comment:
agree, not a usual solution but this is how CN-CBOR works. We can move to other CBOR lib but I don't know whether it supports customizable memory handling.